inventory-management/src/endpoints/router.js

```diff
@@ -0,0 +1,146 @@
+    */
+    routePost(server) {
+        server.route([
+            {
+                method: 'POST',
+                path: `/inventory`,
+                handler: this.handler.addProduct.bind(this.handler),
+                config: {
+                    description: `Add a product to the inventory.`,
+                    tags: ['api', 'inventory'],
+                    plugins: {
+                        'hapi-swagger': {
+                            201: { description: 'Product added to the inventory' },
+                            400: { description: 'Bad request (e.g. body empty)' }
+                        }
+                    }
+                }
+            }
+        ]);
+    }
+
+    /**
+    * All HTTP PUT route definitions
+    * @param {Hapi.server} server - the Hapi server to which to add the route
+    */
+    routePut(server) {
+        server.route([
+            {
+                method: 'PUT',
+                path: '/inventory/{id}',
+                handler: this.handler.updateProduct.bind(this.handler),
+                config: {
+                    description: 'Change the details of product in the inventory.',
+                    tags: ['api', 'inventory'],
+                    plugins: {
+                        'hapi-swagger': {
+                            200: { description: 'Product details updated' },
+                            400: { description: 'Bad request' }
+                        }
+                    }
+                }
+            }
+        ]);
+    }
+
+    /**
+    * All HTTP GET route definitions
+    * @param {Hapi.server} server - the Hapi server to which to add the route
+    */
+    routeGet(server) {
+        server.route([
+            {
+                method: 'GET',
+                path: `/inventory`,
+                handler: this.handler.getProducts.bind(this.handler),
+                config: {
+                    description: 'Get all products in the inventory.',
+                    tags: ['api', 'inventory'],
+                    plugins: {
+                        'hapi-swagger': {
+                            200: { description: 'All Products in inventory returned' },
+                            400: { description: 'Bad request' }
+                        }
+                    }
+                }
+            },
+            {
+                method: 'GET',
+                path: `/inventory/{id}`,
+                handler: this.handler.getProduct.bind(this.handler),
+                config: {
+                    description: 'Retrieve a product from the inventory.',
+                    tags: ['api', 'inventory'],
+                    plugins: {
+                        'hapi-swagger': {
+                            200: { description: 'Product with given ID returned.' },
+                            400: { description: 'Bad request' }
+                        }
+                    }
+                }
```

Comment:
Like @DarshitChanpura said regarding the cart microservice -- it's better to have one method per route, that makes it easier to add/remove routes later.